generator/src/test/java/com/scottlogic/deg/generator/cucumber/features/operators/string/LongerThan.feature

```diff
@@ -284,6 +277,67 @@ Feature: User can specify that a string length is longer than, a specified numbe
       | "U10000XVGZA3" |
       | "twelvedigits" |
 
+  Scenario: 'longerThan' alongside a non-contradicting 'aValid' SEDOL constraint should be successful
+    Given foo is longer than 6
+    And foo is a valid "SEDOL"
+    And foo is in set:
+      | "0263494" |
+      | "3091357" |
+      | "string7" |
+    Then the following data should be generated:
+      | foo       |
+      | null      |
+      | "0263494" |
+      | "3091357" |
+
+  Scenario: 'longerThan' alongside a non-contradicting not 'aValid' SEDOL constraint should be successful
+    Given foo is longer than 2
+    And foo is anything but a valid "SEDOL"
+    And foo is in set:
+      | "0263499" |
+      | "3091352" |
+      | "string7" |
+    Then the following data should be generated:
+      | foo       |
+      | null      |
+      | "0263499" |
+      | "3091352" |
+      | "string7" |
+
+  Scenario: Not 'longerThan' alongside a non-contradicting 'aValid' SEDOL constraint should be successful
+    Given foo is anything but longer than 7
+    And foo is a valid "SEDOL"
+    And foo is in set:
+      | "0263494" |
+      | "3091357" |
+      | "2"       |
+    Then the following data should be generated:
+      | foo       |
+      | null      |
+      | "0263494" |
```

Comment:
shouldn't these be producing 000263494 or GB000263494 ?

Reply:
No, because they're not valid SEDOLs